database/server.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express')
+const app = express()
+const cors = require('cors')
+const pgp = require('pg-promise')()
+const bodyParser = require('body-parser')
+
+const { Todos } = require('./db.js')
+
+app.use( bodyParser.json() );
+app.use( bodyParser.urlencoded({ extended: false }) );
+app.use( cors() );
+
+app.get('/', function(request, response) {
+  Todos.getAll()
+    .then( (results) => response.json(results) )
+})
+
+app.delete('/:id', function (request, response) {
+  const id = request.params.id
+  Todos.deleteOne(id)
+    .then( () => response.json({1: 'success'}) )
+})
+
+app.post('/', function (request, response) {
+  const {task} = request.body
+  Todos.createTask(task)
+    .then( () => response.json({1: 'posted'}) )
+})
+
+app.put('/priority', function (request, response) {
```

Comment:
I've never used PUT before... I've always used POST to update a database before. I'm wondering why you used PUT vs POST